boa/contracts/base_evm_contract.py

```diff
@@ -102,7 +102,10 @@ def __str__(self):
         frame = self.stack_trace.last_frame
         if hasattr(frame, "vm_error"):
             err = frame.vm_error
-            err.args = (frame.pretty_vm_reason, *err.args[1:])
+            if not hasattr(err, "_boa_patched__"):
```

Comment:
might be confusing to future maintainers since this is also used elsewhere to see if the state database has been monkey patched

Reply:
updated in master b6904a9e77b637b6ef2d83c9391e97f1be1dd3ac